scripts/launchCFGVisualizer.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#
+# This file simply redirects cfg-related arguments 
+# to Checker Framework to generate
+#
+
+JSR308=$(cd $(dirname "$0")/../.. && pwd)
```

Comment:
As discussed, please in a separate PR first try to extract the common setup components of these scripts.